build/marketplace/Interfaces.sol

```diff
@@ -5,3 +5,66 @@ pragma solidity 0.8.13;
 interface IErc20 {
     function approve(address, uint256) external returns (bool);
 }
+
+interface IErc5095 {
+    event Redeem(address, address, uint256);
+    event Mature(uint256);
+
+    function maturity() external returns (uint256);
+
+    function underlying() external returns (address);
+
+    function maturityRate() external returns (uint256);
+
+    function convertToUnderlying(uint256) external returns (uint256);
+
+    function convertToPrincipal(uint256) external returns (uint256);
+
+    function maxRedeem(address) external returns (uint256);
+
+    function previewRedeem(uint256) external returns (uint256);
+
+    function maxWithdraw(address) external returns (uint256);
+
+    function previewWithdraw(uint256) external returns (uint256);
+
+    function withdraw(
+        uint256,
+        address,
+        address
+    ) external returns (uint256);
+
+    function redeem(
+        uint256,
+        address,
+        address
+    ) external returns (uint256);
+}
+
+interface IPool {
+    function PT() external returns (IErc5095);
```

Comment:
oh man i hate dealing with abbreviations... `PT` and `SellPT`. I guess that's the interface exposed by the pool tho so, it is what it is. 

generally we only capitalize the First like `Erc20` vs `ERC20` -- our rule (same in our golang code) being only capitalize what you absolute must vs some vague rule about capitalizing the first letter of whatever this abbreviates or whatever

anyway, if the pool is doing it we are stuck with it
all good